kong.conf.default

```diff
@@ -1951,3 +1951,15 @@
                         #
                         # Granularity can be adjusted through the `log_level`
                         # directive.
+
+
+#------------------------------------------------------------------------------
+# WASM
+#------------------------------------------------------------------------------
+#
+#
```

Comment:
```suggestion
```

I couldn't help but notice... (I established `kong.conf.default` and its format) - most other (older) blocks have:
```
#------------------------------------------------------------------------------
# BLOCK
#------------------------------------------------------------------------------

# Additional context

#foo=bar
```

Some blocks (at least the GUI one and this new one) have what I think are editor comment auto-completion:
```
#------------------------------------------------------------------------------
# BLOCK
#------------------------------------------------------------------------------
#
# Additional context
#
#foo=bar
```

Reply:
https://github.com/Kong/kong/pull/11249